test/sym/binop_i32.wast

```diff
@@ -0,0 +1,31 @@
+(module
+  (import "symbolic" "i32_symbol" (func $i32_symbol (result i32)))
+
+  (func $start
+    (local $x i32)
+    (local $res i32)
+    
+    (local.set $x (call $i32_symbol))
+
+    (local.set $res (i32.add (local.get $x) (i32.const 42)))
+    (if (i32.eqz (local.get $res))
+      (then unreachable))
```

Comment:
actually, this could probably be rewritten as:

```wat
(i32.add (local.get $x) (i32.const 42))
i32.eqz
(if (then unreachable))
```

To avoid having a `res` variable.